pkg/etcd/etcd.go

```diff
@@ -140,6 +146,25 @@ func (c CDCEtcdClient) GetAllCDCInfo(ctx context.Context) ([]*mvccpb.KeyValue, e
 	return resp.Kvs, nil
 }
 
+// CheckCDCCluster
+func (c CDCEtcdClient) CheckMultipleCDCClusterExist(ctx context.Context) error {
+	resp, err := c.Client.Get(ctx, BaseKey(""),
+		clientv3.WithPrefix(),
+		clientv3.WithPrevKV())
```

Comment:
1. why use `clientv3.WithPrevKV()`
2. use add optin `clientv3.WithKeysOnly())`? since we only need keys

Reply:
It's a typo and yes, it should be `clientv3.WithKeysOnly()`. I will be careful next time :)